### PR TITLE
Using Bootstrap

### PR DESCRIPTION
**Pull request by BigWizzo**
# HTML and CSS: Using Bootstrap

*  This project clones the Newsweek Magazine website using the Bootstrap  Framework.

* The goal was to use Bootstrap CSS classes wherever it is possible.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,1546 @@
...
+        <button type="submit" class="d-none d-lg-block btn-login">
+          LOGIN</button>&nbsp;&nbsp;
+        <i class="d-lg-none fas fa-user-alt"></i>
+        <button type="button" class="btn btn-primary btn-sm bg-warning border-0 nav-subscribe mx-3">
+          SUBSCRIBE >
+        </button><i class="d-lg-none d-xl-none fas fa-bars"></i>
+      </div>
+    </div>
+  </nav>
+  <!--  -->
```
Comment (kevcap) [inline index.html]: - [x] Please delete **commented piece of codes** in your HTML and CSS files. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 2]
```diff
@@ -0,0 +1,1546 @@
...
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row align-items-center justify-content-center section-topic">
+      <!-- <div class="col-"> -->
+      <span class="n-log">N</span>
+      <h2 class="topic-h2 mb-0">IN THE MAGAZINE</h2>
+      <!-- </div> -->
```
Comment (kevcap) [inline index.html]: - [x] Please delete **commented piece of codes** in your HTML and CSS files. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 3]
```diff
@@ -0,0 +1,1546 @@
...
+          </h5>
+        </div>
+      </div>
+      <!-- </div> -->
+    </div>
+    <div class="row align-items-center justify-content-center section-topic">
+      <!-- <div class="col-"> -->
+      <span class="n-log">N</span>
+      <h2 class="topic-h2 mb-0">EDITORS PICK</h2>
+      <!-- </div> -->
```
Comment (kevcap) [inline index.html]: - [x] Please delete **commented piece of codes** in your HTML and CSS files. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

File: index.html [hunk 4]
```diff
@@ -0,0 +1,1546 @@
...
+    <div class="row align-items-center justify-content-center section-topic">
+      <!-- <div class="col-"> -->
+      <span class="n-log">N</span>
+      <h2 class="topic-h2 mb-0">IN THE MAGAZINE</h2>
+      <!-- </div> -->
+    </div>
+
+    <!-- *** The "In The Magazine" section starts here *** -->
+    <div class="row in-the-magazine">
+      <!-- <div class="row"> -->
```
Comment (kevcap) [inline index.html]: - [x] Please delete **commented piece of codes** in your HTML and CSS files. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

  
#### Hey team, Is a pleasure to review such a good project! Congrats for the amazing work you guys already did, the project looks really nice and I can see that you guys are working hard on it, and we're almost done, there are just some minor changes that you guys have to implement to proceed:

## [REQUIRED CHANGES]
#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
- [x] All screens (Large, Medium, and Small) should have a fixed navbar.
- [x] That image is too shrieked, please fix that.
- Yours
image
- Original
image

- [x] These side images are too small, please take care of it! (Medium and Small screens)
- Yours
image
- Original
image

- [x] In small screens your subscribe section looks broken, please take care of it. ⤵
image

#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵
- [x] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** these repeated rules listed below:
      18 |   font-weight: bold;
       6 |   color: white;

- [x] Please delete **commented piece of codes** in your HTML and CSS files. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))  
:x: **commented piece of codes** /* `.nav { color: red;}` */  
:heavy_check_mark: **comments about code** /*Navigation */  


## Please make the required changes and resubmit for a new code review once you are ready💪

## As always, I wish you all the very best and see you both at the top🚀

  👨🏻‍💻 **TSE: Kevin Alves**

    

  📫 Have any question? Better call Kevin:

  <a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

    
  <sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave::wave::wave: Great work team!

Things to notice:

:100: Using for most part semantic HTML. See observations
:100: descriptive pull request and commit messages
:100: The website is starting to look awesome.
:100: Project has main sections

You've worked on most of the previous required changes, there are still some the project could improve.

- [x]  Links to social network Broken
image

- [x]  Issues and license links broken in README
image

- [x] navbar is missing menu and search input
image

- [x] navbar doesn't match original height
image

- [x] overlaping section titles in most of the views
image
image
image

- [x] Subscribe section not displaying correctly
image

- [x] Navbar should be wrapper around ```header``` HTML tag.

## Keep the great work :muscle:, and Happy Coding :smiley: :computer:!

**Review by ghost — CHANGES_REQUESTED** [review]
# Status: Changes Required

### Hello team,

**Good job on your project. There are still a few fixes required.**

# Mandatory

1). **The navbar section is not displaying properly at 992px width.**

original | clone
--------- | -------
newsweek_original |  newsweek_clone

2). **The circled section of the original webpage is missing on all screen sizes of your webpage. Please add it as it is part of the page and not an advert.**
(The screenshot below is for the full screen size.)

sign_up_edited

# Optional

- For your readme section, you can display your github, twitter usernames instead of @githubhandle and @twitterhandle. Example, Github: @BigWizzo. 

## Good luck!😃.

**PS: You can reach me on slack (Rose Wanjohi)**

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @bigwizzo AND @410AngelaVU 
Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.

- [x] Manage the flow of `My Turn`, `More stories`, `Top Stories`, `Feature Stories`, using the bootstrap classes. For better understanding, you can visit [newsweek](https://www.newsweek.com/)
### Subscribe
- [x] Add two more images
- [x] text should be overflow on images on small screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://github.com/BigWizzo/newsweek-clone/tree/working-branch)|
|:---:|:---:|
|source-page|clone|



##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## Great job working on the previous required changes! 

There are no pending changes.

## :fireworks: You may now merge to master, Congrats!!! :clap: :clap: :clap:

**Pull request merged**